.github/workflows/build-ci-image.yml

```
@@ -0,0 +1,159 @@
+name: Build and Publish CI Docker Image
+
+on:
+  push:
+    branches:
+      - main
```

```suggestion
      - master
```
Oh I missed this. I think this should be master?

---

So I created a main branch.. it's where I deleted everything from the master branch... I think what we will do is when we are ready to switch over, we will switch the default branch for the repo from `master` to `main`... and let `master` be the v1 stuff